examples/wait-for-file-download/README.md

```diff
@@ -2,8 +2,8 @@
 
 **Test Code**: [test.js](test.js)
 
-This example shows how to specify that a test should to wait for the file to download.
+This example shows how to specify that a test should wait for the file to download.
```

Comment:
In the future, don't combine the different changes for two examples into one PR.

Reply:
ok